### ISSUE TITLE
Naming/PredicateMethod false positive with explicit type conversion methods

### ISSUE THREAD

**Issue by ntkme** [rubocop/rubocop#14242]
## Expected behavior

It shouldn't complain method name like `to_b`, `to_bool`, which is usually an intentional type conversion method.

## Actual behavior

```
lib/sass/value.rb:41:9: C: Naming/PredicateMethod: Predicate method names should end with ?.
    def to_bool
        ^^^^^^^
```

## Steps to reproduce the problem

- Create a method named `to_b` or `to_bool` etc that returns a boolean

## RuboCop version

```
$ bundle exec rubocop -V
1.76.0 (using Parser 3.3.8.0, rubocop-ast 1.45.0, analyzing as Ruby 3.1, running on ruby 3.4.4) [arm64-darwin24]
  - rubocop-performance 1.25.0
  - rubocop-rake 0.7.1
  - rubocop-rspec 3.6.0
```

**Comment by ntkme** [comment]
Maybe we can have a predefined exclude list like `[:to_b, :to_bool, :to_boolean]`? Not sure if it would be good enough or not.

**Comment by dvandersluis** [comment]
The cop has an `AllowedMethods` configuration that you can use to list method names that should not register offenses. Please see https://docs.rubocop.org/rubocop/1.76/cops_naming.html#namingpredicatemethod

**Comment by koic** [comment]
Yeah, whether this should be provided by default is questionable, but users can at least configure it as follows.

```yaml
# .rubocop.yml
Naming/PredicateMethod:
  AllowedMethods:
    - to_b
    - to_bool
    - to_boolean
```

**Comment by ntkme** [comment]
Thanks. Didn't know that the list already exist.
